easybuild/tools/options.py

```diff
@@ -215,12 +215,16 @@ def __init__(self, *args, **kwargs):
         # set up constants to seed into config files parser, by section
         self.go_cfg_constants = {
             self.DEFAULTSECT: {
-                'DEFAULT_REPOSITORYPATH': (self.default_repositorypath[0], "Default easyconfigs repository path"),
+                'DEFAULT_REPOSITORYPATH': (self.default_repositorypath[0], 
+                                           "Default easyconfigs repository path"),
                 'DEFAULT_ROBOT_PATHS': (os.pathsep.join(self.default_robot_paths),
                                         "List of default robot paths ('%s'-separated)" % os.pathsep),
+                'USER': (os.environ['USER'], "Current username, derived from the os environment"),
+                'HOME': (os.environ['HOME'], 
```

Comment:
Sounds ok to me, but in practice this should be the same as `$USER`, right?

So, I would check the templated value against `$USER` in the tests, which is a good sanity check